### DISCUSSION TITLE
Reading app version returns nothing

### DISCUSSION THREAD

**Discussion by Demonic098** [darthcloud/BlueRetro#141] [Q&A]
Hi.
Reading app version returns nothing when it want open `BlueRetro OTA FW update`
![2022-01-15-141400_430x934_scrot](https://user-images.githubusercontent.com/25936174/149621546-12d81eb2-ed25-4d2d-ac3b-00e19e8e2661.png)
.
What can be dune to fix it ?
using latest BlueRetro commit aa3054f5d15f49b72016581e55443e0ad0eff1b2
with latest RetroRoot

**Comment by darthcloud** [comment]
Are you sure your programmed FW is latest?

Only way I can see for the version to now show is if the FW do not contain the version characteristics on the BLE interface. Which was FW < v1.0.

If you use an official build does it work?

**Reply by Demonic098** [reply]
> 

yes

**Reply by Demonic098** [reply]
I guess i know why its not detecting version number 
i've modified monitor baud rate (my serial to USB do not support speed more than 460800)
and now in monitor version returns  
`# bt_att_cmd_app_ver_rsp v1.4-dirty`
and web script cannot process -dirty
